var/spack/repos/builtin/packages/must/package.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Must(CMakePackage):
+    """MUST detects usage errors of the Message Passing Interface (MPI)
+    and reports them to the user. As MPI calls are complex and usage
+    errors common, this functionality is extremely helpful for application
+    developers that want to develop correct MPI applications. This includes
+    errors that already manifest: segmentation faults or incorrect results
+    as well as many errors that are not visible to the application developer
+    or do not manifest on a certain system or MPI implementation."""
+
+    homepage = "https://www.i12.rwth-aachen.de/go/id/nrbe"
+    url = "https://hpc.rwth-aachen.de/must/files/MUST-v1.8.0-rc1.tar.gz"
+
+    maintainers = ["jgalarowicz", "dmont"]
+
+    version(
+        "1.8.0-rc1",
+        sha512="ea30291ff5de2a43b225561ce65c61863a7afc5a127af0cc7b149d77574308981290a0913e1aa5547d88739bf7d56ae35cb29d7d164b175958ca4542f319da5b",
+    )
+    version(
+        "1.8-preview", sha256="67b4b061db7a893e22a6610e2085072716d11738bc6cc3cb3ffd60d6833e8bad"
+    )
+    version(
+        "1.7.2",
+        sha256="c6a106a694a29bfb494e6bf5ced55853acf16994abc350e13ee5e0e6f24df601666fe60531e59d784acd82874682385c479271fbe04e56b927d8d1054cc035b2",
```

Comment:
Do you want to make `1.7.2` the `preferred` (or stable) version since the newer ones are `preview` and `rc1`?  (See https://spack.readthedocs.io/en/latest/packaging_guide.html#version-selection.)

Reply:
Hi Tamara - we prefer the 1.8.0 rc1 version because the others are too old and contain code that is now deprecated.
Thanks for the review and the sha256 values.  I added them and submitted the changes.  Thanks again!